docs/topics/xml-rule-when-syntax.adoc

```diff
@@ -535,3 +536,89 @@ The `has-classification` element is used to determine if a specified classificat
 |String
 |An optional title to match the classification against.
 |====
+
+[[graph-query-syntax]]
+== <graph-query> Syntax
+
+=== Summary
+
+Use the `<graph-query>` element to search the generated graph for any elements. This element is primarily used to determine if specific classes are embedded within an archive, such as a JAR or EAR. For a better understanding of the `<graph-query>` condition, see the JavaDoc for the http://windup.github.io/windup/docs/latest/javadoc/org/jboss/windup/config/condition/GraphCondition.html[GraphCondition] class. 
```

Comment:
Perhaps say that this is to search for specific archives instead of specific classes in an archive? The general intent here is to search for dependent libraries by name, rather than based upon anything about their content.